GPU/GLES/ShaderManagerGLES.cpp

```diff
@@ -371,7 +371,7 @@ static inline bool GuessVRDrawingHUD(bool is2D, bool flatScreen) {
 	//HUD cannot have full texture alpha
 	else if (gstate_c.textureFullAlpha) hud = false;
 	//HUD must have full vertex alpha
-	else if (!gstate_c.vertexFullAlpha) hud = false;
+	else if (!gstate_c.vertexFullAlpha && gstate.getDepthTestFunction() == GE_COMP_NEVER) hud = false;
```

Comment:
I'm surprised it uses NEVER?  That would only write stencil, I guess.

-[Unknown]

Reply:
In Assassin's Creed there are 2D black rectangles outside of viewport. The rectangles are only visible if I scale the coordinate down into the viewport (using HUD scaling).

No idea what is it good for.